target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -190,6 +190,7 @@ ath79_setup_interfaces()
 		;;
 	buffalo,wzr-hp-g300nh-rb|\
 	buffalo,wzr-hp-g300nh-s|\
+	trendnet,tew-673gru|\
```

Comment:
Please maintain alphabetical order

Reply:
Thanks for catching that (and apologies you needed to).